source/includes/kubernetes/_k8_daemonsets.md

```diff
@@ -76,3 +76,73 @@ Retrieve a daemon set and all its info in a given [environment](#administration-
 | `status`<br/>_object_                      | The status information of the daemon set                        |
 
 Note that the list is not complete, since it is refering to the [kubernetes api details](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md).
+
+<!-------------------- CREATE DAEMON SET -------------------->
+
+### Create a daemon set
+```shell
+curl -X POST \
+  -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/pods"
```

Comment:
Should this endpoint be `daemonsets`?

Reply:
yessir 😳

Reply:
Done! :D cc94a073d874ada178c6c92399ae0e3921f9dcb5